src/EasySpeech.js

```diff
@@ -645,6 +645,9 @@ EasySpeech.speak = ({ text, voice, pitch, rate, volume, force, infiniteResume, .
   if (!validate.text(text)) {
     throw new Error('EasySpeech: at least some valid text is required to speak')
   }
+  if ((new TextEncoder().encode(text)).length > 4096) {
+    throw new Error('EasySpeech: text exceeds max length of 4096 bytes.')
+  }
```

Comment:
This could be a breaking change for some users, who don't use the google voices.
I know it's tricky to bind it to specific voices as you'll never know which voices fail, but it bring some trouble .

Reply:
Good point. However, the problem is we can't be sure if a google voice contains "google" in it's name. For example on Android the voices have cryptic names and you can't even say by the name, whether this is even a male or female voice.

Therefore, I think trying to detect google voices brings much bigger trouble than this one.
I would also then make this a major version bump and add to the changelog that it's breaking.

Reply:
@stereonom would you see an alternative? Raising a warning instead?

Reply:
@jankapunkt Oh, I think that would be much better than breaking it for all users. <3